src/spv/lift.rs

```diff
@@ -1378,12 +1378,11 @@ impl Module {
 
         // IDs can be allocated once we have the full sets needing them, whether
         // sorted by contents, or ordered by the first occurence in the module.
-        let mut id_bound = NonZeroU32::new(1).unwrap();
+        let mut id_bound = NonZeroU32::MIN;
```

Comment:
This is actually fine because the values don't really matter. You can even write `spv::Id::MIN` if you want to get cheeky with it, heh.